src/client/store/actions/audioActions.test.js

```diff
@@ -0,0 +1,81 @@
+import {
+  CREATE_AUDIO_DATA,
+  START_CREATION_AUDIO_DATA,
+  PLAY_PAUSE_SOUND_FROM_FILE,
+  CREATE_STREAME_DATA,
+  START_MUTE_STREAME_AUDIO,
+  MERGE_CANVAS_WIDTH,
+} from './types';
+
+import {
+  createAudioData,
+  startCreationAudioData,
+  playPauseSoundFromFile,
+  createStreamData,
+  startMuteStreamAudio,
+  mergeCanvasWidth,
+} from './audioActions';
+
+describe('TEST AUDIO ACTIONS', () => {
+  it('SHOULD CREATE AUDIO DATA', () => {
```

Comment:
It should dispatch action CREATE AUDIO DATA

Reply:
ok